project/ProjectPlugin.scala

```diff
@@ -52,7 +52,7 @@ object ProjectPlugin extends AutoPlugin {
 
     lazy val micrositeSettings: Seq[Def.Setting[_]] = Seq(
       micrositeName := "SBT-Hood",
-      micrositeBaseUrl := "/sbt-hood",
+      micrositeBaseUrl := "/sbt",
```

Comment:
was this intended?

Reply:
Yes it is @BenFradet , It was one of the things the Hood team wanted to change. Now the docs urls will be:
Graddle-Hood: `https://47deg.github.io/hood/gradle/`
SBT-Hood: `https://47deg.github.io/hood/sbt/`
That way keeps coherence.

Reply:
Thanks :+1: didn't know about the `hood` prefix

Reply:
Ey @BenFradet I think I´m missing the `hood` prefix in the `micrositeBaseUrl` setting.
`micrositeBaseUrl := "/hood/sbt",`
I´ll change it in another PR.